.checkpatch.conf

```diff
@@ -8,5 +8,7 @@
 --ignore=BIT_MACRO
 --ignore=PREFER_PRINTF
 --ignore=PREFER_SCANF
+--ignore=VOLATILE
```

Comment:
I'm not sure why these commits got attached to this pull request. I'll need to sort that out with Maxim when he gets back. The pull should only be for the feature.h and related init changes.